### PR TITLE
[WIP] Remove ApMon-CPP dependency from Monitoring recipe

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

**Comment by awegrzyn** [comment]
@ktf Can I ask you to do usual fixes?

**Comment by Barthelemy** [comment]
I have made the changes , let's see if it pleases the linter

**Comment by ktf** [comment]
How do you plan to support Monalisa in Monitoring?

File: monitoring.sh [hunk 1]
```diff
@@ -4,7 +4,6 @@ tag: v2.6.3
 requires:
   - boost
   - "GCC-Toolchain:(?!osx)"
-  - ApMon-CPP
```
Comment (ktf) [inline monitoring.sh]: Maybe we can start by dropping it in the `o2-dataflow` default?

**Comment by awegrzyn** [comment]
Indeed, let's wait for WP8 meeting to discuss it

**Comment by ktf** [comment]
For the record, @Atlantic777 is doing some fix in #1381 which should allow supporting newer Redhat flavoured distributions.

**Pull request closed without merging**